xknx/knxip/hpai.py

```diff
@@ -21,6 +21,11 @@ def __init__(self, ip_addr: str = "0.0.0.0", port: int = 0):
         self.ip_addr = ip_addr
         self.port = port
 
+    @property
+    def route_back(self) -> bool:
+        """Return True if HPAI is a route back address information."""
+        return self.ip_addr == "0.0.0.0"
```

Comment:
```suggestion
```

I don't think we need this. We should not add properties that are not used anywhere (except tests).

Reply:
Tests need to be adjusted as well